examples/client/client.c

```diff
@@ -3007,6 +3007,16 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
 
     showPeerEx(ssl, lng_index);
 
+#ifdef HAVE_OCSP
+    {
+        struct tm tm;
+        char date[32];
+        ret = wolfSSL_get_ocsp_producedDate_tm(ssl, &tm);
+        if ((ret == 0) && (strftime(date, sizeof date, "%Y-%m-%d %H:%M:%S %z",&tm) > 0))
```

Comment:
Does `time.h` need to be included? For portability we could add an autoconf detection of `strftime` in config.h by adding it to `configure.ac` `AC_CHECK_FUNCS`.

Reply:
That's a really good point.  I've done that, and also added fallback `!HAVE_STRFTIME` code that uses `wolfSSL_get_ocsp_producedDate()`.

Aside: remarkably, autotools will let you test for the existence of a type, or a _member_ of a named struct, but there's no built-in way to just test that a struct exists.  Really odd.